lib/rules/no-side-effects.js

```diff
@@ -103,12 +119,25 @@ module.exports = {
         'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-side-effects.md',
     },
     fixable: null,
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          ignoreClosures: {
+            type: 'boolean',
+            default: false,
+          },
+        },
+      },
+    ],
   },
 
   ERROR_MESSAGE,
 
   create(context) {
+    // Options:
+    const ignoreClosures = context.options[0] && context.options[0].ignoreClosures;
```

Comment:
Can you use `context.options[0]?.ignoreClosures ?? false`?

Reply:
That would be nice, but optional chaining is a Node 14 feature, and this plugin still supports Node 10.